src/translate.js

```diff
@@ -1,14 +1,14 @@
 import curry from 'lodash.curry';
 import { connect } from 'react-redux';
 import { getP } from './selectors';
-import { isFunction, isString } from './private/utils';
+import { isFunction, isString, isObject } from './private/utils';
 
 const getDisplayName = Component => (
     Component.displayName || Component.name || 'Component'
 );
 
-const mapPolyglotToProps = polyglotScope => state => ({
```

Comment:
I disagree, it takes the `p` object and bind it to the props, not the options.
It can also be called `mapPToProps`
Anyway, as this is a factory, we can name it `createMapPolyglotToProps` or `makeMapPolyglotToProps`